Product/Production/Adapters/CORE_X12DocSubmission_a0/src/main/java/gov/hhs/fha/nhinc/corex12/ds/realtime/adapter/AdapterX12RealTimeImpl.java

```diff
@@ -35,34 +34,40 @@
 
 /**
  *
- * @author sadusumilli
+ * @author sadusumilli, cmay
  */
-public class AdapterCORE_X12DSRealTimeImpl extends BaseService {
+public class AdapterX12RealTimeImpl extends BaseService {
 
     /**
      *
-     * @param adapterRequest
+     * @param request the adapter request
```

Comment:
You'll find a lot of this across my PR.  I clarified a few items, but I didn't have time to add documentation where there wasn't any before.  We should revisit this after 4.6 because yes, we should appropriately document input, output, and functionality.